specification/security/resource-manager/Microsoft.Security/preview/2024-02-01-preview/assessmentMetadata.json

```diff
@@ -0,0 +1,1224 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Defender for Cloud",
```

Comment:
Can you please add the prevvious versions of these 2 files in the first commit so that we canreview the diff across the versions ?

Reply:
Hi, I am not sure I follow - both `assessments.json` and `assessmentMetadata.json` are included in the first commit as a copy from the 202without any changes, with their previous version date (`2021-06-01`). Appreciate your help

Reply:
you need to paste it in the same folder otherwise , we cannot do a diff. 

Your first commit is this one : https://github.com/Azure/azure-rest-api-specs/pull/27389/commits/e4f6045fe7851b06e4d251a804d149428fcb7cb7 and you created the files under the "2023-04-01-preview" folder 

THe current review is for the 2024-02-01-preview folder. The first commit should be with the previous version in the same folder , this enables reviewers to diff the changes across commits..

For now you can just create a new commit in the same folder and then overwrite it with another commit and let me know th commit id so that i know which to diff from.,